04_distributed.ipynb

```diff
@@ -16,9 +16,7 @@
     "\t* [Making a cluster](#Making-a-cluster)\n",
     "\t\t* [Detailed method](#Detailed-method)\n",
     "\t\t* [Simple method](#Simple-method)\n",
-    "\t* [Executing with the distributed client](#Executing-with-the-distributed-client)\n",
-    "\t* [Distributed futures](#Distributed-futures)\n",
```

Comment:
I used the Calysto plugin to generate TOC. There were no actual sections with these titles, so it removed it from TOC. I think those are useful topics, but they are not covered in the current material.